docs/contributing/release_process.rst

```diff
@@ -159,7 +159,8 @@ Wagtail 1.4 will still support only Django 1.8 (LTS) and 1.9.
 Release process
 ===============
 
-Wagtail uses a time-based release schedule, with feature releases every two months or so.
+Wagtail uses a time-based release schedule, with feature releases every 
+`three months <https://github.com/wagtail/wagtail/wiki/Release-schedule>`_.
```

Comment:
@tom I think the link might be better applied to `time-based release schedule` in this paragraph

Reply:
Thanks, @ababic, I agree.